src/fdk_rdf_parser/reference_data/extend_dataset.py

```diff
@@ -48,39 +48,43 @@ def extend_distributions(
     open_licenses: Optional[Dict[str, SkosCode]],
     media_types: Optional[Dict[str, SkosCode]],
 ) -> Optional[List[Distribution]]:
-    if distributions is None or open_licenses is None:
+    if distributions is None:
         return distributions
     else:
         extended_distributions = []
         for dist in distributions:
-            if dist.license is not None:
-                extended_licenses = []
-                for lic in dist.license:
-                    ref_code = (
-                        open_licenses.get(remove_trailing_slash(lic.uri))
-                        if lic.uri is not None
-                        else None
-                    )
-                    if ref_code is not None:
-                        dist.openLicense = True
-                        if lic.prefLabel is None:
-                            lic.prefLabel = ref_code.prefLabel
-                    extended_licenses.append(lic)
-                dist.license = extended_licenses
-            if dist.format is not None:
-                extended_mediatypes = []
-                for fmt in dist.format:
-                    ref_code = (
-                        media_types.get(fmt.lower()) or
-                        media_types.get("text/" + fmt.lower()) or
-                        media_types.get("application/" + fmt.lower()) or
-                        media_types.get("application/vnd." + fmt.lower())
-                        if fmt is not None
-                        else None
-                    )
-                    if ref_code is not None:
-                        extended_mediatypes.append(ref_code)
-                dist.mediaType = extended_mediatypes
+            if open_licenses is not None:
```

Comment:
Her er det vel mer en bare "syntax fix". Må også legge til tester for å ha coverage 100%.

Reply:
Ja holder på. Sleit litt med set og rekkefølge men har løst det på en annen måte.